Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -30,16 +30,19 @@ configuration:
   name: template
   required: false
   type: 0
-- display: Use URL Filtering for auto enrichment
+- additionalinfo: "add here"
```

Comment:
:ma:

Reply:
@kirbles19 will review, more data in the slack thread referenced in the issue

Reply:
Okay cool